src/lib/html.ts

```diff
@@ -21,29 +44,25 @@ export function generateHtml(
     webtreemapCss: btoa(fs.readFileSync(require.resolve('./vendor/webtreemap.css'))),
   };
 
-  // Create a combined bundle if applicable
-  if (exploreResults.length > 1) {
-    exploreResults = [makeMergedBundle(exploreResults)].concat(exploreResults);
-  }
+  const treeDataMap = getTreeDataMap(exploreResults);
 
   // Get bundles info to generate select element
-  const bundles = exploreResults.map((data) => ({
+  let bundles = exploreResults.map((data) => ({
     name: data.bundleName,
     size: formatBytes(data.totalBytes),
   }));
 
-  // Get webtreemap data to update map on bundle select
-  const treeDataMap = exploreResults.reduce<Record<string, { name: string; data: WebTreeMapNode }>>(
-    (result, data, index) => {
-      result[index] = {
-        name: data.bundleName,
-        data: getWebTreeMapData(data.files),
-      };
+  // Create a combined bundle if applicable
+  if (exploreResults.length > 1) {
+    bundles = [
+      {
+        name: COMBINED_BUNDLE_NAME,
+        size: formatBytes(exploreResults.reduce((total, result) => total + result.totalBytes, 0)),
+      },
+      ...bundles,
+    ];
```

Comment:
Oh wait, I think it's better if we keep `bundles` as a `const`, so these line could be:

```suggestion
    bundles.unshift({
      name: COMBINED_BUNDLE_NAME,
      size: formatBytes(exploreResults.reduce((total, result) => total + result.totalBytes, 0)),
    });
```

Reply:
I'll apply this change in next commits

Reply:
Hey @nikolay-borzov , can you release this PR to npm? It's been a while since the last release, and I really want this to rollout my next release. (And don't forget to update the suggestion above). Many thanks.

Reply:
That's weird. I don't remember why I decided not to release right when this PR was merged. I will take care of it this within this week

Reply:
Thanks.

I think you wanted to release this PR on 2.5.0 that you're working on it, right? That's why you added this to the milestone.

Reply:
Yeah. And for some reason, I thought #185 will be easy to do